proto/tx.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package proto;
+
+message TxMsg {
+    string id = 1;
```

Comment:
For the `InitialReport` this will be "" so maybe you want to just put this in the `Report`?